faker/providers/ssn/nl_BE/__init__.py

```diff
@@ -0,0 +1,57 @@
+# coding=utf-8
+
+from __future__ import unicode_literals
+from .. import Provider as SsnProvider
+from faker.generator import random
+from faker.providers.date_time import Provider as DateProvider
+from faker.providers import BaseProvider
+
+"""
+For more info on rijksregisternummer, see https://nl.wikipedia.org/wiki/Rijksregisternummer
+Dutch/French only for now ...
+"""
+
+class Provider(SsnProvider):
+
+    @classmethod
+    def ssn(cls):
```

Comment:
could you add some tests about this? I'd like to both test people born before and after the year 2000